mautic/mautic
# Add Redis to DDEV

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Add Redis for DDEV - your local enviroment based on 


- https://github.com/drud/ddev-contrib/tree/master/docker-compose-services/redis
- https://github.com/drud/ddev-contrib/tree/master/docker-compose-services/redis-commander





#### Steps to test this PR:


1. Load up [this PR](https://mautibox.com)
2. For new installation run `ddev start` 
3. For already exist ddev installation run `ddev restart`
4. After build you should see Redis Commander on https://mautic.ddev.site:1359/
![image](https://user-images.githubusercontent.com/462477/137639555-c16574ee-106a-4254-96f0-7cf3f638a1d4.png)

5. Your redis instance is available inside the containers at `ddev-mautic-redis:6379`




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10538"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
